ui/shared/components/form/Inputs.jsx

```diff
@@ -21,11 +22,12 @@ export class BaseSemanticInput extends React.Component {
       if (nextProps.options.length !== (options || []).length) {
```

Comment:
Could this section (line 21-30) be replaced with lodash isEqual?
Or should this comparator be extracted and reused on the other Input components?

``if (_.isEqual(options, nextProps.options)) return true``

Reply:
Yes I expect _.isEqual() to give the same behaviour as the custom comparator. If we prefer to use the custom comparator in seqr, rather than adding a library/dependency, then I think the comparator should be extracted to a utility so it can be reused for other components that need it.

Reply:
All my use of lodash here has been removed from this feature. 👍